packages/api/src/tasks/checkAlerts.ts

```diff
@@ -262,18 +262,26 @@ export const handleSendGenericWebhook = async (
     'Content-Type': 'application/json', // default, will be overwritten if user has set otherwise
     ...(webhook.headers?.toJSON() ?? {}),
   };
-
   // BODY
-
   let body = '';
-  if (webhook.body) {
-    const handlebars = Handlebars.create();
-    body = handlebars.compile(webhook.body, {
-      noEscape: true,
-    })({
-      body: escapeJsonString(message.body),
-      link: escapeJsonString(message.hdxLink),
-      title: escapeJsonString(message.title),
+  try {
+    if (webhook.body) {
```

Comment:
this if case is probably redundant? given the `compile` would throw

Reply:
yep, remove this part.